### ISSUE TITLE
Can't upload album arts, it says I ran out of disk space

### ISSUE THREAD

**Issue by ivellios1988** [AzuraCast/AzuraCast#5893]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

v0.17.5 Stable

### What happened?

Recently I've been working a lot with adding album arts to my media files. I did it using MP3Tag. When I uploaded the updated MP3s to my VPS and logged in to AzuraCast, I noticed that these files lack any album arts, there's just a 'no image' icon instead, just like the pictures didn't exist.

When I checked my station's '.albumart' directory it turned out that there appeared a ton of empty JPEGs.

Then I tried to upload some JPEG's via AzuraCast dashboard and I saw an error messsage indicating I ran out of disk space.

![Bez tytułu](https://user-images.githubusercontent.com/13646017/204657110-2b01f2a2-9b73-4082-a94d-9fb1b97d9a11.png)

Which I find surprising since I have a lot of free space on my VPS (30 gigabytes on system disk + some 5 gigabytes on a secondary disk which serves as my station's media storage).

Later I tried to upload some files into the '.albumart' directory via SSH and everything was fine, there was no indication that I may run out of disk space.

### Relevant log output

```shell
[2022-11-29T21:46:10.796187+00:00] AzuraCast.ERROR: Album Artwork for "/var/azuracast/stations/radio_paranormalium/media/audycje/Radio Paranormalium i Portal Infra - Debaty (nie)kontrolowane/XI Debata (nie)kontrolowana (31 lip 2016) Ooparts, czyli artefakty spoza swoich czasow.mp3" could not be processed: "Unable to write file at location: .albumart/84e5bae10e81c3afc2bb1451.jpg. file_put_contents(): Write of 247287 bytes failed with errno=28 No space left on device" [{"file":"/var/azuracast/www/vendor/league/flysystem/src/Local/LocalFilesystemAdapter.php","line":120,"function":"atLocation","class":"League\\Flysystem\\UnableToWriteFile","type":"::"},{"file":"/var/azuracast/www/vendor/league/flysystem/src/Local/LocalFilesystemAdapter.php","line":98,"function":"writeToFile","class":"League\\Flysystem\\Local\\LocalFilesystemAdapter","type":"->"},{"file":"/var/azuracast/www/vendor/league/flysystem/src/Filesystem.php","line":57,"function":"write","class":"League\\Flysystem\\Local\\LocalFilesystemAdapter","type":"->"},{"file":"/var/azuracast/www/src/Entity/Repository/StationMediaRepository.php","line":250,"function":"write","class":"League\\Flysystem\\Filesystem","type":"->"},{"file":"/var/azuracast/www/src/Entity/Repository/StationMediaRepository.php","line":195,"function":"writeAlbumArt","class":"App\\Entity\\Repository\\StationMediaRepository","type":"->"},{"file":"/var/azuracast/www/src/Media/MediaProcessor.php","line":178,"function":"loadFromFile","class":"App\\Entity\\Repository\\StationMediaRepository","type":"->"},{"file":"/var/azuracast/www/src/Flysystem/LocalFilesystem.php","line":76,"function":"App\\Media\\{closure}","class":"App\\Media\\MediaProcessor","type":"->"},{"file":"/var/azuracast/www/src/Media/MediaProcessor.php","line":179,"function":"withLocalFile","class":"App\\Flysystem\\LocalFilesystem","type":"->"},{"file":"/var/azuracast/www/src/Media/MediaProcessor.php","line":38,"function":"processMedia","class":"App\\Media\\MediaProcessor","type":"->"},{"file":"/var/azuracast/www_tmp/CompiledContainer.php","line":473,"function":"__invoke","class":"App\\Media\\MediaProcessor","type":"->"},{"file":"/var/azuracast/www/vendor/symfony/messenger/Middleware/HandleMessageMiddleware.php","line":95,"function":"{closure}","class":"CompiledContainer","type":"::"},{"file":"/var/azuracast/www/src/MessageQueue/HandleUniqueMiddleware.php","line":41,"function":"handle","class":"Symfony\\Component\\Messenger\\Middleware\\HandleMessageMiddleware","type":"->"},{"file":"/var/azuracast/www/vendor/symfony/messenger/Middleware/SendMessageMiddleware.php","line":71,"function":"handle","class":"App\\MessageQueue\\HandleUniqueMiddleware","type":"->"},{"file":"/var/azuracast/www/vendor/symfony/messenger/MessageBus.php","line":73,"function":"handle","class":"Symfony\\Component\\Messenger\\Middleware\\SendMessageMiddleware","type":"->"},{"file":"/var/azuracast/www/vendor/symfony/messenger/Worker.php","line":156,"function":"dispatch","class":"Symfony\\Component\\Messenger\\MessageBus","type":"->"},{"file":"/var/azuracast/www/vendor/symfony/messenger/Worker.php","line":105,"function":"handleMessage","class":"Symfony\\Component\\Messenger\\Worker","type":"->"},{"file":"/var/azuracast/www/src/Console/Command/MessageQueue/ProcessCommand.php","line":87,"function":"run","class":"Symfony\\Component\\Messenger\\Worker","type":"->"},{"file":"/var/azuracast/www/vendor/symfony/console/Command/Command.php","line":308,"function":"execute","class":"App\\Console\\Command\\MessageQueue\\ProcessCommand","type":"->"},{"file":"/var/azuracast/www/vendor/symfony/console/Application.php","line":1032,"function":"run","class":"Symfony\\Component\\Console\\Command\\Command","type":"->"},{"file":"/var/azuracast/www/vendor/symfony/console/Application.php","line":301,"function":"doRunCommand","class":"Symfony\\Component\\Console\\Application","type":"->"},{"file":"/var/azuracast/www/vendor/symfony/console/Application.php","line":171,"function":"doRun","class":"Symfony\\Component\\Console\\Application","type":"->"},{"file":"/var/azuracast/www/bin/console","line":16,"function":"run","class":"Symfony\\Component\\Console\\Application","type":"->"}] []
```

**Comment by GaetanSsfm** [comment]
If someone as a command for increase data storage ?

**Comment by ivellios1988** [comment]
I only have one radio station and it doesn't have any storage limits. As I said, I have plenty of free disk space, and for some reason I can't upload any cover arts - it creates files that have size of 0 bytes and throws error messages.

**Comment by ivellios1988** [comment]
Oh, wait, wait... looks like, indeed, there's something wrong with my partitions :|

/dev/sdb1 shows size 183G, used 169G, so in theory there should be 14 gigs available but it only shows 4.6G. Any idea how to fix this?

```
df -lh
Filesystem      Size  Used Avail Use% Mounted on
udev            953M     0  953M   0% /dev
tmpfs           194M  1.1M  193M   1% /run
/dev/sda1        39G   16G   24G  39% /
tmpfs           970M     0  970M   0% /dev/shm
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           970M     0  970M   0% /sys/fs/cgroup
/dev/sda15      105M  5.2M  100M   5% /boot/efi
/dev/sdb1       183G  169G  4.6G  98% /var/lib/docker/volumes/azuracast_station_data/_data/radio_paranormalium/media
/dev/loop2       92M   92M     0 100% /snap/lxd/23991
/dev/loop1       50M   50M     0 100% /snap/snapd/17883
/dev/loop3       92M   92M     0 100% /snap/lxd/24061
/dev/loop5       64M   64M     0 100% /snap/core20/1738
/dev/loop0       56M   56M     0 100% /snap/core18/2654
/dev/loop4       56M   56M     0 100% /snap/core18/2667
/dev/loop6       64M   64M     0 100% /snap/core20/1778
tmpfs           194M     0  194M   0% /run/user/0
overlay          39G   16G   24G  39% /var/lib/docker/overlay2/ac900a8044303a30eb5047da83efd7215114ea8beef4cc023554329fe927d30a/merged
```

**Comment by freqq68** [comment]
@ivellios1988 go to my git : https://github.com/AzuraCast/AzuraCast/issues/5909#issuecomment-1340259538 

I don't have investigated this for the moment caus' I don't have free time :/ 

But I hope that I help you!
